Correct spelling in cah_us

domcleal opened the pull request:
And remove now-duplicated entries from cah_us_uk complementary pack.

grifferz commented:
Removing lines from the YAML files is not currently supported unfortunately. It will result in every card after the removed one being shifted down by one in everyone's hands, and probably a crash when the maximum possible card index is dealt to someone.

Could you possibly split this in to a pull request for the typo fixes and one for the removal? (or teach me how to split it in two myself?)

The removals could then be merged after I've worked out how to handle that.

domcleal committed:
Correct spellings in US packs

Duplicates will be removed later.

domcleal commented:
Sure, this PR now just fixes the typos.

To fix it, I did:
1. `git reset --soft HEAD^` to bring the commit back into the index
2. `git reset packs/cah_us_uk.yml && git co packs/cah_us_uk.yml` to reset that file
3. fixed spellings in cah_us_uk.yml
4. `git ci -a`
5. `git push -f github`

Then to create the other branch, I rebased the original commit I did here onto the updated one:
1. `git co -b patch-4-removal 4d469c6`
2. `git rebase --onto patch-4 HEAD^`
3. fixed the conflict in the yaml file
4. `git add packs/cah_us_uk.yml`
5. `git rebase --continue`
